### PR TITLE
IN-611-[SDK]-Editor enhance test coverage

### PR DESCRIPTION
**Pull request by bhavin-te**
# Description

This PR contains code-changes for IN-611- Enhanced Test Coverage for Editor[SDK].

Module : `tenable\io\editor.py `
Statements covered by Test-cases (%) : 93%

Added test cases to cover maximum statement of Editor module. Earlier test cases covered 88% statement of editor module. Now 93% statement is covered.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

To test code-changes of this PR, I have run testcases of `pyTenable\tests\io\test_editor.py`. All testcases of this file working fine and getting passed. please find attached snapshot.

image

Also I have run all test-cases of `pyTenable\tests `and it is getting passed. please find attached snapshot.

image

**Test Configuration**:
* Python Version(s) Tested:
* Tenable.sc version (if necessary):

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by MitaliUpadhyay — APPROVED** [review]

**Review by alpesh-te — APPROVED** [review]
LGMT!

File: tests/io/test_editor.py [hunk 1]
```diff
@@ -170,3 +171,46 @@ def test_editor_details(api):
...
+def test_editor_parse_plugins_response(api):
+    '''test to parse_plugins method'''
+    families = {
+        0: {'count': 11459, 'id': 28, 'name': 'AIX Local Security Checks', 'status': 'mixed'},
+        1: {'count': 360, 'id': 30, 'name': 'Alma Linux Local Security Checks', 'status': 'mixed'},
+        2: {'count': 60, 'id': 27, 'name': 'Policy Compliance', 'status': 'mixed'}
+    }
+    endpoint_obj = APIEndpoint(api)
+
+    api.editor.parse_plugins(
```
Comment (MitaliUpadhyay) [inline tests/io/test_editor.py:213]: Please correct indention issue for api.editor.parse_plugins().

File: tests/io/test_editor.py [hunk 2]
```diff
@@ -170,3 +171,46 @@ def test_editor_details(api):
     for each_scan_id in scan_ids_list:
         api.scans.delete(each_scan_id)
 
+
+@pytest.mark.vcr()
+def test_editor_parse_plugins_id_keyerror(api):
```
Comment (MitaliUpadhyay) [inline tests/io/test_editor.py:176]: Testcase name should be `test_editor_parse_plugins_family_id_keyerror` or `test_editor_parse_plugins_family_id_keyerror` ?

Comment (MitaliUpadhyay) [inline tests/io/test_editor.py:176]: Testcase name should be `test_editor_parse_plugins_id_keyerror` or `test_editor_parse_plugins_family_id_keyerror` ?

**Review by varap11 — APPROVED** [review]

**Pull request merged**